### PR TITLE
Implement customise language display name

### PR DESCRIPTION
**Pull request by aeng**
https://bugzilla.redhat.com/show_bug.cgi?id=1169662

### CODE REVIEW

**Comment by huangp** [comment]
:+1:

**Comment by aeng** [comment]
@zanata-jenkins retest this please

**Comment by aeng** [comment]
@zanata-jenkins retest this please

File: zanata-war/src/main/resources/db/changelogs/db.changelog-3.7.xml [hunk 1]
```diff
@@ -13,4 +13,18 @@
     </addColumn>
   </changeSet>
 
+  <changeSet id="2" author="aeng@redhat.com">
+    <comment>Add displayName, nativeName column to HLocale</comment>
+    <addColumn tableName="HLocale">
+      <column name="displayName" type="varchar(100)">
+        <constraints nullable="true" />
+      </column>H
```
Comment (djansen-redhat) [inline zanata-war/src/main/resources/db/changelogs/db.changelog-3.7.xml]: ?

**Comment by djansen-redhat** [comment]
Do we *need* a tooltip, or something, to indicate the clearing the fields reverts to the default value?

**Comment by aeng** [comment]
Placeholder text indicated the default value if field leave empty. Its not showed on the first load due to seam issue.

**Comment by djansen-redhat** [comment]
It appears that when the name has been changed, then removed, the item shows up without a label on the Languages list page.

**Comment by djansen-redhat** [comment]
- [ ] Changing the name, but not the native name, sometimes causes an exception

Also, do you think this change effectively invalidates [1035068 Improve the Admin Add language feature](https://bugzilla.redhat.com/show_bug.cgi?id=1035068)?  I'd go with "no" as the fields are not available at the time of create, and the behaviour is subtly different - but being able to customise the display name at all gives the user an option.

**Comment by djansen-redhat** [comment]
:white_check_mark: Tested

**Comment by aeng** [comment]
Merged into integration/master. https://github.com/zanata/zanata-server/commit/81054b713c3d9b1e1962915188c8d40c6513c0b5

**Pull request closed without merging**